docs/en_US/sqls/lexical_elements.md

```diff
@@ -76,3 +76,22 @@ Example, `SELECT TRUE AS field1 FROM demo` , the field `field1` always returns `
 ```text
 DD, HH, MI, SS, MS
 ```
+
+**String Literals**:
+
+```text
+"user", 'user'
+```
+
+Please note that when using single quotes for string literals in the command line, you may encounter the following issue:
```

Comment:
Also give a solution, like suggest to use \"abc\"

Reply:
ok, I will fix it.